### PR TITLE
add Arabic Language

### PR DESCRIPTION
**Pull request by Tayeb-Ali**
> **Describe your change**
> 
>  Added Arabic language.
> 

> **New feature checklist**
> 
> * [x]  Have you run the application to make sure your code runs
> * [x]  Is this not just a simple formatting change
> 
> **Translation checklist**
> 
> * [x]  Have you run the application to make sure your code runs
> * [x]  Have you used the correct keys for your translations added (e.g. `en`)
> * [x]  Have you made sure to leave a `,` at the end of each translation added
> * [x]  Have you added the language to `supportedLanguages` in alphabetical order
> * [x]  Have you added the language to the table in README.md in alphabetical order
> * [ ]  If you have translated the README, have you linked it in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ If you're having issues with the packaged executable or using this tool in gener
 
 | Language                                    | Code  | Translator                                    | Translated                           |
 | ------------------------------------------- | ----- | --------------------------------------------- | ------------------------------------ |
+| العربية | ar | [Tayeb-Ali](https://github.com/Tayeb-Ali) | UI |
```
Comment (brentvollebregt) [inline README.md:106]: Could we make the language column "Arabic (العربية)"? Also could you pad the colums out to match the others?

Just going for consistency 👍

File: auto_py_to_exe/web/js/i18n.js [hunk 2]
```diff
@@ -1151,6 +1217,10 @@ const getTranslation = (path) => {
 
 const _defaultLanguage = 'en';
 const supportedLanguages = [
+    {
+        name: 'العربية',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:1221]: Can this be "Arabic (العربية)"?

File: auto_py_to_exe/web/js/i18n.js [hunk 3]
```diff
@@ -416,6 +441,7 @@ const translationMap = {
                 id: 'Konversi .py ke .exe',
                 cz: 'Převést .py na .exe',
                 gr: 'Μετατροπή .py σε .exe',
+                ar: 'تحويل .py إلى. exe'
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:444]: Can you add a comma after this line

File: auto_py_to_exe/web/js/i18n.js [hunk 4]
```diff
@@ -622,6 +658,9 @@ const translationMap = {
                 gr: 'Να είσατε προσεκτικοί με την χρήση επιλογής ενός αρχείου\n' +
                     '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">read this</a>\n' +
                     'and update your code to work with PyInstaller.',
+                    ar: 'كن حذرًا عند استخدام ملفات إضافية مع وضع الملف الواحد ؛ \n'+
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:661]:
```suggestion
                ar: 'كن حذرًا عند استخدام ملفات إضافية مع وضع الملف الواحد ؛ \n' +
```

File: auto_py_to_exe/web/js/i18n.js [hunk 5]
```diff
@@ -637,6 +676,7 @@ const translationMap = {
                 id: 'Jika Anda ingin menyimpan berkas di direktori root, tambahkan titik (.) di tujuan.',
                 cz: 'Pokud chcete přidat soubory do kořenové složky, přidejte tečku (.) do destinace.',
                 gr: 'Έαν θέλετε να εισάγετε τα αρχεία σας στον κεντρικό φάκελο απλά τοποθετήστε μια τελεία (.) στον προορισμό',
+                ar: 'إذا كنت تريد وضع الملفات في الدليل الجذر ، فضع نقطة (.) في الوجهة.'
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:679]:
```suggestion
                ar: 'إذا كنت تريد وضع الملفات في الدليل الجذر ، فضع نقطة (.) في الوجهة.',

```

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
Looking good - just a few asks. Thank you!

File: README.md [hunk 6]
```diff
@@ -103,6 +103,7 @@ If you're having issues with the packaged executable or using this tool in gener
 
 | Language                                    | Code  | Translator                                    | Translated                           |
 | ------------------------------------------- | ----- | --------------------------------------------- | ------------------------------------ |
+| Arabic (العربية)                              | ar   | [Tayeb-Ali](https://github.com/tayeb-ali)      | UI                                   |
```
Comment (brentvollebregt) [inline README.md:106]: Looks like this might need to be shifted over a bit more

File: auto_py_to_exe/web/js/i18n.js [hunk 7]
```diff
@@ -637,6 +676,8 @@ const translationMap = {
                 id: 'Jika Anda ingin menyimpan berkas di direktori root, tambahkan titik (.) di tujuan.',
                 cz: 'Pokud chcete přidat soubory do kořenové složky, přidejte tečku (.) do destinace.',
                 gr: 'Έαν θέλετε να εισάγετε τα αρχεία σας στον κεντρικό φάκελο απλά τοποθετήστε μια τελεία (.) στον προορισμό',
+                ar: 'إذا كنت تريد وضع الملفات في الدليل الجذر ، فضع نقطة (.) في الوجهة.',
+
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:680]: Can we remove this redundant line?

**Review by brentvollebregt — COMMENTED** [review]
Looking good - just a couple more things regarding consistency

File: README.md (hunk 6 above)
Comment (brentvollebregt) [reply README.md]: @Tayeb-Ali it looks like the changes you made didn't align this one up

Comment (Tayeb-Ali) [reply README.md]: > @Tayeb-Ali it looks like the changes you made didn't align this one up
It was fixed in the last update below this comment.
Commit ID = [d73ed60](https://github.com/brentvollebregt/auto-py-to-exe/pull/319/commits/d73ed6011c6b776c538077699ac2aae5c21316f9)

Comment (brentvollebregt) [reply README.md]: These were the changes in d73ed6011c6b776c538077699ac2aae5c21316f9

```diff
  | Language                                    | Code  | Translator                                    | Translated                           |
  | ------------------------------------------- | ----- | --------------------------------------------- | ------------------------------------ |
- | Arabic (العربية)                              | ar   | [Tayeb-Ali](https://github.com/tayeb-ali)      | UI                                   | 
+ | Arabic (العربية) 	| ar 	| [ Tayeb-Ali ](https://github.com/tayeb-ali) 	| UI |
  | Brazilian Portuguese (Português Brasileiro) | pt_br | [marleyas](https://github.com/marleyas)       | UI (partial)                         
  |
  | Chinese Simplified (简体中文)               | zh    | [jiangzhe11](https://github.com/jiangzhe11)   | UI and [README](./README-Chinese.md) |
  | Chinese Traditional (繁體中文)              | zh_tw | [startgo](https://github.com/ystartgo)        | UI                                   |
```

See how the `|`'s are lining up like the others?

**Comment by brentvollebregt** [comment]
This has been manually fixed up and added in 78229678920f6dd33c53d5f560bd5f627a506efd

**Pull request closed without merging**